content/en/real_user_monitoring/guide/rum-for-product-analytics.md

```diff
@@ -0,0 +1,376 @@
+                            "sort": {
+                                "aggregation": "count",
+                                "order": "desc"
+                            }
+                        }
+                    ]
+                }
+            ],
+            "formulas": [
+                {
+                    "formula": "query1",
+                    "limit": {
+                        "count": 10,
+                        "order": "desc"
+                    }
+                }
+            ]
+        }
+    ]
+}
+{{< /code-block >}}
+
+To investigate which users are clicking on these buttons, modify the search query by selecting the **Table** visualization type and clicking **+** to add another `group` field for `@user.name`.
+
+{{< img src="real_user_monitoring/guide/rum-for-product-analytics/actions_by_user_name_in_cart_page.png" alt="Search query for actions grouped by user name on Shopist's Cart page" style="width:90%;">}}
+
+For example:
+
+{{< code-block lang="json" filename="json" disable_copy="false" collapsible="true" >}}
+{
+    "viz": "query_table",
+    "requests": [
+        {
+            "response_format": "scalar",
+            "queries": [
+                {
+                    "data_source": "rum",
+                    "name": "query1",
+                    "indexes": [
+                        "*"
+                    ],
+                    "compute": {
+                        "aggregation": "count"
+                    },
+                    "search": {
+                        "query": "@type:action @view.name:\"/cart\""
+                    },
+                    "group_by": [
+                        {
+                            "facet": "@action.name",
+                            "limit": 5,
+                            "sort": {
+                                "aggregation": "count",
+                                "order": "desc"
+                            }
+                        },
+                        {
+                            "facet": "@usr.name",
+                            "limit": 5,
+                            "sort": {
+                                "aggregation": "count",
+                                "order": "desc"
+                            }
+                        }
+                    ]
+                }
+            ],
+            "formulas": [
+                {
+                    "formula": "query1",
+                    "limit": {
+                        "count": 25,
+                        "order": "desc"
+                    }
+                }
+            ]
+        }
+    ],
+    "has_search_bar": "auto"
+}
+{{< /code-block >}}
+
+## Analyze the conversion rate of core workflows
+
+Use the [funnel visualization type][6] to track the conversion rate across crucial areas in your website. 
+
+Once you have created a funnel based on views and actions on your website, you can do the following actions:
```

Comment:
and, the last action on line 156 is confusing bc we are talking about actions in terms of the funnel earlier and then things we are saying. can we use a different ending word?

Reply:
```suggestion
Once you have created a funnel based on views and actions on your website, you can use it in the following ways:
```